__tests__/reducers/create-user-reducer.js

```diff
@@ -0,0 +1,11 @@
+import {getUserInitialState} from '../../lib/reducers/create-user-reducer'
+import {restoreDateNowBehavior, setDefaultTestTime} from '../test-utils'
+
+describe('lib > reducers > create-user-reducer', () => {
+  afterEach(restoreDateNowBehavior)
+
+  it('should be able to create the initial state', () => {
+    setDefaultTestTime()
+    expect(getUserInitialState({}, {})).toMatchSnapshot()
```

Comment:
I don't think this method `getUserInitialState` has two args.

Reply:
Removed in https://github.com/opentripplanner/otp-react-redux/pull/329/commits/b7b608d2c9c8a57b9bbcbe28b81a59ef0b41a77e.